ada_2_fhir-r4/zibs2020/ada_project_drivers/_driverInclude.xsl

```diff
@@ -258,15 +258,15 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
             <xsl:when test="$localName = 'apgar_score'">
                 <xsl:apply-templates select="$in" mode="nl-core-ApgarScore">
                     <xsl:with-param name="subject" select="$subject"/>
-                    <xsl:with-param name="profile" select="'nl-core-ApgarScore-1Minute'"/>
+                    <xsl:with-param name="profile" select="$profileNameApgarScore1Minute"/>
```

Comment:
The benefit is to have one single place where the profile name is defined, instead of the need to always type it manually, with the risk of making typos. Moreover, while typing a variable, oXygen gives suggestions (making it even easier to add the correct one), and it raises a validation error if a variable is not defined.